### DISCUSSION TITLE
mods

### DISCUSSION THREAD

**Discussion by spearminttt** [headlesshq/headlessmc#82] [General]
can it/does it launch ur currently used minecraft profile with all your mods and the mod settings?

**Comment by 3arthqu4ke** [comment]
I am not sure what you mean by minecraft profile. Do you mean your account?
HeadlessMc will have no problem launching e.g. forge or fabric with all the mods in the mods folder.

**Reply by spearminttt** [reply]
so it will launch my currently used minecraft installation with all the installed mods?

**Reply by 3arthqu4ke** [reply]
It can and will launch what is in the .minecraft folder (the directory the game will be running in and consequently the mods folder with the mods can also be configured by the `hmc.gamedir` property). The mods will likely manage their settings themselves.

 It will **not** take into account profiles set in the official Minecraft launcher. There you can also configure which directory the game is running in, launch arguments etc. but HeadlessMc will not read those configurations.
